custom/icds/data_management/templates/data_management/data_management.html

```diff
@@ -6,22 +6,69 @@
 {% requirejs_main 'data_management/js/data_management' %}
 
 {% block page_content %}
-<div class="container">
-  <div class="row">
-    <div class="page-header">
-      <h1>
-        {% trans "Data Management" %}
-      </h1>
-    </div>
+{% registerurl "paginate_data_management_requests" domain %}
+
+<div class="page-header">
+  <h1>
+    {% trans "Data Management" %}
+  </h1>
+</div>
+<div class="panel panel-modern-gray panel-form-only">
+  <div class="panel-body">
+    {% crispy form %}
   </div>
-  <div class="row">
-    <div>
-      <div class="panel panel-modern-gray panel-form-only">
-        <div class="panel-body">
-          {% crispy form %}
-        </div>
+</div>
+
+<div class="panel panel-default ko-template" id="data-management-requests-panel">
+    <div class="panel-heading">
+      <h3 class="panel-title">{% trans 'Data Management Requests' %}</h3>
+    </div>
+    <div class="panel-body">
+      <div class="alert alert-danger" data-bind="visible: error, text: error"></div>
+      <div data-bind="visible: showLoadingSpinner">
+        <i class="fa fa-spin fa-spinner"></i>
+        {% trans "Loading requests..." %}
+      </div>
+      <div data-bind="visible: noRequestsMessage">
+        <p class="alert alert-info">
```

Comment:
nitpick: just put the alert & alert-info classes on the div, you don't need a paragraph